src/SDKs/Search/DataPlane/Search.Tests/Utilities/EncryptionFixture.cs

```diff
@@ -0,0 +1,224 @@
+            Provider provider = client.Providers.Register("Microsoft.KeyVault");
+        }
+
+        private AccessPolicyEntry CreateReadOnlyAccess(string securityPrincipalId)
+        {
+            return new AccessPolicyEntry
+            {
+                TenantId = Guid.Parse(TestEnvironmentFactory.GetTestEnvironment().Tenant),
+                ObjectId = securityPrincipalId,
+                Permissions = new Permissions()
+                {
+                    Keys = new List<string>()
+                    {
+                        KeyPermissions.Get,
+                        KeyPermissions.WrapKey,
+                        KeyPermissions.UnwrapKey,
+                    }
+                }
+            };
+        }
+
+        private AccessPolicyEntry CreateReadAndWriteAccess(string securityPrincipalId)
+        {
+            return new AccessPolicyEntry
+            {
+                TenantId = Guid.Parse(TestEnvironmentFactory.GetTestEnvironment().Tenant),
+                ObjectId = securityPrincipalId,
+                Permissions = new Permissions()
+                {
+                    Keys = new List<string>()
+                    {
+                         KeyPermissions.Get,
+                        KeyPermissions.WrapKey,
+                        KeyPermissions.UnwrapKey,
+                        KeyPermissions.Create,
+                        KeyPermissions.Delete,
+                        KeyPermissions.Get,
+                        KeyPermissions.List
+                    }
+                }
+            };
+        }
+
+        private Vault CreateKeyVault(List<AccessPolicyEntry> accessPolicies)
+        {
+            KeyVaultManagementClient keyVaultMgmtClient = this.MockContext.GetServiceClient<KeyVaultManagementClient>();
+            string keyVaultName = TestUtilities.GenerateName("keyvault");
+            return keyVaultMgmtClient.Vaults.CreateOrUpdate(ResourceGroupName, keyVaultName, new VaultCreateOrUpdateParameters
+            {
+                Location = "northcentralus",
+                Properties = new VaultProperties
+                {
+                    TenantId = Guid.Parse(TestEnvironmentFactory.GetTestEnvironment().Tenant),
+                    AccessPolicies = accessPolicies,
+                    Sku = new Management.KeyVault.Models.Sku(SkuName.Standard)
+                }
+            });
+        }
+
+        private KeyBundle CreateKeyVaultKey(Vault keyVault)
+        {
+            string keyName = TestUtilities.GenerateName("keyvaultkey");
+
+            KeyVaultClient keyVaultClient = GetKeyVaultClient();
+
+            int keySize = 2048;
+            return keyVaultClient.CreateKeyAsync(
+                keyVault.Properties.VaultUri, 
+                keyName, 
+                JsonWebKeyType.Rsa, 
+                keySize,
+                JsonWebKeyOperation.AllOperations, 
+                new KeyAttributes()
+           ).GetAwaiter().GetResult();
```

Comment:
Missing `ConfigureAwait(false)`? I'm not actually sure if it's needed in a test context; just double-checking